output.go

```diff
@@ -341,6 +385,8 @@ func (o *OutputOptions) GetOutput() (io.Writer, error) {
 	return o.outputMulti, nil
 }
 
+// GetError returns a Writer that has the stderr output from the process that
+// the OutputOptions that this method is called on is attached to.
```

Comment:
I'd just have this say something like "returns a writer that can be used for standard error, according to the output configuration.